### DISCUSSION TITLE
Unable to accept the answer

### DISCUSSION THREAD

**Discussion by AnirudhG07** [yetone/avante.nvim#1002] [Q&A]
Hey! thanks for the cool plugin!
I am unable to accept answer when I asked on chat. I am using copilot

Steps to reproduce
1) Open Empty chat
2) Ask, write a function in python for checking if a number is a prime number or not.
3) press "A" to accept the answer but it just shifted the focus on the code without the code being added

I tried `leader a`, `leader A`, nothing worked. How to add the code from avante to my main code buffer?

Here are my [dotfiles](https://github.com/AnirudhG07/dotfiles/blob/stow/nvim/.config/nvim/lua/anirudh/plugins/avante.lua) for reference.

Some more observations:
1) I saw some YouTube videos and they see `<a> apple this, <A> to apply all` something on top of the code in avant buffer, I DONT see that. Rather I see nothing.
So probably is it like some keybinding conflict?? How do I resolve that?(if thats the case)

```
Replace lines: 1-0
```python
def prime(n):
    """Check if a number is a prime number."""
    if n <= 1:
        return False
    for i in range(2, int(n**0.5) + 1):
        if n % i == 0:
            return False
    return True

def test_prime():
    """Test cases for the prime function."""
    assert prime(2) == True, "Test case 1 failed"
    assert prime(3) == True, "Test case 2 failed"
    assert prime(4) == False, "Test case 3 failed"
    assert prime(5) == True, "Test case 4 failed"
    assert prime(11) == True, "Test case 5 failed"
    
 ```
 This is what I see

**Comment by nagashimam** [comment]
This happens to me, too. Seems you have to write "..., and apply the code to the file" or something like it at the end of your prompt.

I haven't figured out workarounds for key bindings such as leader a not working, though. Looks explicitly overriding key bindings to ones without leader may work
